neural_lam/models/ar_model.py

```diff
@@ -476,6 +534,11 @@ def plot_examples(self, batch, n_examples, prediction=None):
                 ]
 
                 example_i = self.plotted_examples
+                # for i, fig in enumerate(var_figs):
+                #     fn = f"example_{i}_{example_i}_t{t_i}.png"
+                #     fig.savefig(fn)
+                #     logger.info(f"Saved example plot to {fn}")
+
```

Comment:
Remove or uncomment

Reply:
:) I will remove, I was trying to sneak it in for future debugging purposes. But it shouldn't be here... :)